manoseimas/lobbyists/views/lobbyists.py

```
@@ -0,0 +1,30 @@
+# -*-coding: utf-8 -*-
+
+from django.shortcuts import render
+from manoseimas.lobbyists.models import Lobbyist
+
+def lobbyist_list(request):
+    """A placeholder."""
+    lobbyists = Lobbyist.objects.all()
+    return render(request, 'lobbyist_list.jade', {lobbyists: lobbyists})
+
+
+def lobbyist_profile(request, lobbyist_slug):
+    """A profile view for a lobbyist."""
+
+    try:
+        lobbyist = Lobbyist.objects.get(slug=lobbyist_slug)
+    except Lobbyist.DoesNotExist:
+        lobbyist = None
```

Why not return a 404 error here?  Why show an empty page with a fake name?